test/frontend/test_node.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import numpy as np
+import tensorflow as tf
+
+from onnx_tf.frontend import convert_graph
+from onnx import helper
+
+# for testing
+from onnx_tf.backend import prepare
+
+class TestNode(unittest.TestCase):
+  """ Tests for nodes
+  """
+  def _get_rnd(self, shape, low=-1.0, high=1.0):
+    return np.random.uniform(low, high, np.prod(shape)) \
+                    .reshape(shape) \
+                    .astype(np.float32)
+
+  def test_relu(self):
+    shape = (10, 10)
+    x = tf.placeholder(tf.float32, shape=shape)
+    y = tf.nn.relu(x)
+
+    tf_graph = y.graph.as_graph_def(add_shapes=True)
+    for node in tf_graph.node:
+        if node.name == "Relu":
```

Comment:
Can we make this more robust or find a way of getting the output info directly from y?